requirements.txt

```diff
@@ -21,3 +22,8 @@ pytest-ordering
 sphinx
 sphinx-autobuild
 sphinx_rtd_theme
+
+#GCE
```

Comment:
Please add these dependencies to `setup.py` as well. (section `install_requires`, alphabetic sort)